bin/gaffer.cmd

```diff
@@ -147,6 +147,22 @@ if "%ONNX_ROOT%" NEQ "" (
 	call :appendToPath "%ONNX_ROOT%\lib" PATH
 )
 
+rem RenderMan
+rem =========
+
+if "%RMANTREE%" NEQ "" (
+	if "%RMANTREE:~-1%" EQU "\" (
+		set RMANTREE_STRIPPED=%RMANTREE:~0,-1%
```

Comment:
Does this create an environment variable called `RMANTREE_STRIPPED`? If so, might be worth avoiding?

Reply:
Yes it does, so that's not good. I think we can just let the `\` double-up and still be fine. Done and squashed down to the original commit.